winterfell/webapp/src/js/app.js

```diff
@@ -171,7 +171,7 @@ app.config(['$stateProvider', '$urlRouterProvider', function ($stateProvider, $u
 
   $stateProvider.state({
     name: 'root.claimsubmit',
-    url: '/claim/{claimId}/submit',
+    url: '/claim/{claimId}/submit?error',
```

Comment:
on the submit page, we check if the error is truthy (not undefined or null)
so if there's no query param for `error`, we're ok. There might be a cleaner way to do this though